include/clang/Basic/DiagnosticASTKinds.td

```diff
@@ -355,4 +355,6 @@ def warn_odr_non_type_parameter_type_inconsistent : Warning<
 def err_unsupported_ast_node: Error<"cannot import unsupported AST node %0">;
 def warn_ast_importer_missing_decl_in_decl_context : Warning<
   "missing %0Decl in imported DeclContext <%1:%2:%3>">;
+def warn_ast_importer_merged_decl_in_decl_context : Warning<
+  "merged %0Decl in DeclContext from <%1:%2:%3> to <%4:%5:%6>">;
```

Comment:
I think, we could come up with a better message here. Something like "Different Decls are mapped/imported to the same %0Decl in DeclContext ..."

Reply:
Probably "%0Decl <%1:%2:%3> is imported as existing <%4:%5:%6>" ?

Reply:
I feel that the point is that we map two distinct decls in the "from" context to the same decl in the "to" ctx.

Reply:
This message tells something similar: `"%0Decl <%1:%2:%3> was mapped to <%4:%5:%6> at AST import"`We can see that one decl was mapped to another one (with different location). I could not find other possible short (one line) string. This message may show up now relatively often (at least for LLVM).

Reply:
Ok, sounds good.